arcgis-ios-sdk-samples/Layers/RGB renderer/README.metadata.json

```diff
@@ -37,7 +37,7 @@
     ],
     "snippets": [
         "RGB Renderer Cells.swift",
```

Comment:
😩 Why they used filename with spaces in the first place...

Reply:
Haha I also wondered if I should rename that file. @philium, what do you think?

Reply:
Yes please! In fact, you could just move the classes into `RGBRendererSettingsViewController.swift`. There's not really a reason for them to be in a separate file.